lib/_http_client.js

```diff
@@ -133,10 +133,12 @@ function ClientRequest(options, cb) {
   if (self.socketPath) {
     self._last = true;
     self.shouldKeepAlive = false;
-    const optionsPath = {
-      path: self.socketPath
+    const createConnectionOptions = {
```

Comment:
To me `optionsPath` is more general, like 'options for (unix) socket path' as opposed to 'options for TCP socket'. With that in mind, I'd prefer to see the name unchanged. If other @nodejs/collaborators disagree, at the very least I'd rather see a more succinct name like `connectOptions` or `createOptions` or similar.